src/dotnet/HQ.Abstractions/Staff/UpsertStaffV1.cs

```diff
@@ -23,11 +23,13 @@ public class Request
         public DateOnly? EndDate { get; set; }
         public string? FirstName { get; set; }
         public string? LastName { get; set; }
-        public string? Email { get; set; }
+        public bool? CreateUser { get; set; }
+        public string Email { get; set; } = null!;
     }
 
     public class Response
     {
         public Guid Id { get; set; }
+        public Guid UserId { get; set; }
```

Comment:
We should make this nullable, otherwise it would default to Guid.Empty (all zeros) when serializing the response from the API if the user does not check Create User.